mod-configuration-server/src/main/resources/templates/db_scripts/schema.json

```diff
@@ -4,6 +4,11 @@
       "run": "after",
       "snippetPath": "migration/config_data.sql",
       "fromModuleVersion": "mod-configuration-5.3.0"
+    },
+    {
+      "run": "after",
+      "snippetPath": "migration/migrateConfigDataOtherSettingsNamings.sql",
+      "fromModuleVersion": "mod-configuration-5.8.1"
```

Comment:
Should simply be 5.8.0 , so it's triggered from all versions before 5.8.0. eg 5.7.1.

Reply:
Fixed